tests/tests.js

```diff
@@ -236,6 +237,43 @@ QUnit.test( "useEffect(..)", function test(assert){
 	assert.strictEqual( pActual, pExpected, "call without TNG wrapping context" );
 } );
 
+QUnit.test("useMemo(..)", function test(assert) {
+	function foo(num, rest) {
+		return useMemo(() => {
+			return num;
+		}, rest);
+	}
+
+	function baz(num, rest) {
+		return useMemo(() => {
+			return num * num;
+		}, rest);
+	}
+
+	var pExpected = "error";
```

Comment:
I realize this test suite style is rather unorthodox. It's habit from a long time ago for me. See the other tests, where I use the expected/actual pattern, with like `rExpected` and `rActual`. For reference, I use the peculiar progression of naming prefixes for these pairs of variables of `r`, `p`, `q`, `t`, `s`, `u`, `h`, and `j`.

I am happy to fix up this test accordingly, but if you look at the other tests you should be able to see the pattern if you want to give it a shot.